kpi/utils/data_exports.py

```diff
@@ -79,9 +94,72 @@
         'key': METADATA,
         'columns': USER_FIELDS + METADATA_FIELDS,
     },
+    'access_logs_export': {
+        'queryset': lambda: apps.get_model('audit_log', 'AccessLog').objects.all(),
+        'key': 'metadata',
+        'columns': ACCESS_LOGS_EXPORT_FIELDS,
+    },
 }
 
 
+def create_data_export(
```

Comment:
If we do it this way we'll have to add to this method every time we create a new kind of export task. I think it would make more sense to have this method take just take data and headers and create a csv from it. The AccessLogExportTask and ProjectViewExportTask classes would do whatever they had to do to get the data and then call this method with the data they collected.